ansible/roles/vm_set/files/mux_simulator.py

```diff
@@ -664,11 +666,16 @@ def mux_status(vm_set, port_index):
     Returns:
         object: Return a flask response object.
     """
+    global GET_MUX_COUNTER
     _validate_vm_set(vm_set)
     if not g_muxes.has_mux(port_index):
         abort(404, 'Unknown bridge, vm_set={}, port_index={}'.format(vm_set, port_index))
 
     if request.method == 'GET':
+        GET_MUX_COUNTER += 1
+        if GET_MUX_COUNTER % 100 == 0:
+            app.logger.info('===== {} GET {} with port {} ====='.format(request.remote_addr, request.url, port_index))
```

Comment:
The log message should indicate how many GET requests have been received since the last time a GET request was logged.

Instead of log a message for every 100 GET requests, we should also consider log a message after some fixed time in case the clients are quite for some time.

Reply:
Should we also consider log the response instead of just the request?

Reply:
Fixed